### PR TITLE
Moveable Map Markers Documentation

### PR DESCRIPTION
**Pull request by Tittoffer**
**When merged this pull request will:**

- Update markers.md to include moveable map markers.

### CODE REVIEW

File: docs/wiki/feature/map-gestures.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: wiki
+title: Map Gestures
+description: Allow players to gesture on map for other in close proximity to see.
```
Comment (jonpas) [inline docs/wiki/feature/map-gestures.md]: I'd just keep `to point` for clarity.

File: docs/wiki/feature/map-gestures.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### 1.1 Map gestures
+
+Allow players to gesture/point on their map for others in close proximity to see.
+
+## 2. Usage
+
+### 2.1 Gesturing on map
+
+- To gesture on map press and hold <kbd>Left Mouse Btn</kbd>.
```
Comment (jonpas) [inline docs/wiki/feature/map-gestures.md]: Same.

File: docs/wiki/feature/markers.md [hunk 3]
```diff
@@ -15,8 +15,23 @@ version:
...
+Adds the ability to move map markers after they are placed. (Note: Must be enabled in `Markers` module)
+
+
+## 2. Usage
+
+### 2.1 Moving map markers
+
+- To move a marker hold down <kbd>Alt</kbd> and <kbd>Left Mouse Btn</kbd> on marker and move.
+
+Note: does not work in Editor. Conflict with vanilla teleport.
```
Comment (jonpas) [inline docs/wiki/feature/markers.md]: Use:
```
<div class="panel callout">
    <h5>Note:</h5>
    <p>Does not work in Editor Preview! Conflicts with vanilla teleport keybind.</p>
</div>
```

File: docs/wiki/feature/map-gestures.md (hunk 2 above)
Comment (Tittoffer) [reply docs/wiki/feature/map-gestures.md]: Que?

Comment (jonpas) [reply docs/wiki/feature/map-gestures.md]: As above, `to point`.

**Comment by Tittoffer** [comment]
Should be ready for merge 👍

**Pull request merged**